data-raw/mike_wright_temperature_regression/cvpia_rearing_lengths.csv

```diff
@@ -0,0 +1,93 @@
+order,watershed,lifestage,miles,feet,source,species
```

Comment:
this file is redundant, it comes from `cvpiaHabitat::watershed_lengths`, see where it is used and replace with packaged data plz

also, delete the temps.tiff and lock file from this PR